lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -376,6 +377,13 @@ registerFunction("exp", "n", "n", function(self, args)
 	return exp(rv1)
 end)
 
+registerFunction("frexp", "n", "r", function(self, args)
+	local op1 = args[2]
+	local rv1 = op1[1](self, op1)
+	local mantissa, exponent = frexp(rv1)
+	return { mantissa, exponent }
+end)
+
```

Comment:
there's no point in using `registerFunction` here. The only reason the other functions in this file use it is because this file was created before the `e2function` syntax existed, and it hasn't changed since.